src/main/java/core/basesyntax/impl/Element.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.impl;
+
+public class Element<K, V> {
```

Comment:
Element contains 2 elements inside? It's a little bit confusing. 
I'd recommend renaming this class
```suggestion
public class Pair<K, V> {
```